include/z64scene.h

```diff
@@ -499,8 +496,8 @@ typedef enum {
 #define SCENE_CMD_TIME_SETTINGS(hour, min, speed) \
     { SCENE_CMD_ID_TIME_SETTINGS, 0, CMD_BBBB(hour, min, speed, 0) }
 
-#define SCENE_CMD_SKYBOX_SETTINGS(externalTextureFileId, skyboxId, weather, lightMode) \
-    { SCENE_CMD_ID_SKYBOX_SETTINGS, externalTextureFileId, CMD_BBBB(skyboxId, weather, lightMode, 0) }
+#define SCENE_CMD_SKYBOX_SETTINGS(skyboxId, weather, isIndoors) \
```

Comment:
This isn't really accurate, "indoors" is just a different lighting mode, hence the original name I gave it

Reply:
Does make sense for this value for having a different value other than 0/1?

Reply:
Hmmm the only valid values are 0 and 1 so admittedly I'm not entirely sure, fig worked on decompiling some of this stuff so perhaps he is more knowledgable, but to understanding the "indoor" mode doesn't use directional lights (which is used by the sun and moon) so calling it indoor *technically* makes sense although I don't think it's the best name for this, making it true or false is probably fine but I think we should maybe understand it a little better in order to properly name it.

Reply:
We can always cameback and rename this in the future when we have more knowledge about how this works. So I think this name is fine for now.

Reply:
indoors is perfectly fine for this, ive already named it that in my kankyo branch.

Reply:
you say its not accurate in the first message, if you have any examples of how its not accurate feel free to explain. but based on the code Ive worked with, its a fine description. Its a true or false flag that says whether it should be using the time of day entries to control lighting (outdoors) or use the light settings (indoors). and its used in a couple more places as well

Reply:
> but to understanding the "indoor" mode doesn't use directional lights 
they do use directional lights, but they are not oriented to the position of the sun and moon. the direction and color is indicated by the light settings

Reply:
I believe its used in the exterior of Ganon's Castle, which is not exactly "out"doors, among other places.